qutebrowser/browser/commands.py

```diff
@@ -818,10 +818,13 @@ def paste(self, sel=False, tab=False, bg=False, window=False):
         text = utils.get_clipboard(selection=sel)
         if not text.strip():
             raise cmdexc.CommandError("{} is empty.".format(target))
-        log.misc.debug("{} contained: '{}'".format(target,
-                                                   text.replace('\n', '\\n')))
-        text_urls = enumerate(u for u in text.split('\n') if u.strip())
-        for i, text_url in text_urls:
+        log.misc.debug("{} contained: {!r}".format(target, text))
+        text_urls = [u for u in text.split('\n') if u.strip()]
+        if (len(text_urls) > 1 and not urlutils.is_url(text_urls[0]) and
+            urlutils.get_path_if_valid(
+                text_urls[0], check_exists=True) is None):
```

Comment:
If you wonder why the condition is changed it was beacause it was wrong. I only forgot to mention this in the commit :(